frontend/src/components/playlist.js

```diff
@@ -1,19 +1,254 @@
 import React, { Component } from 'react';
-import logo from '../logo.svg';
+import { Input, Button, Select } from 'antd';
+import { Container, Row, Col } from 'reactstrap';
+import 'antd/dist/antd.css';
+import '../styles/playlist.css'
 import '../App.css';
+import Embedded from './embedded'
+
+
+const { Search, TextArea } = Input;
+const { Option } = Select;
+
+let accessToken = ''
+let Spotify = require('spotify-web-api-js');
+let spotifyApi = new Spotify();
+spotifyApi.setAccessToken(accessToken);
 
 class Playlist extends Component {
+  constructor(props) {
+    super(props);
+    this.state={
+      user: {
+          id: '',
+          playlistId: '',
+          iframe:''
+      },
+      artist: {
+          name: '',
+          id: ''
+      },
+      playlistName: 'Playlist App',
+      playlistDescription: 'Created with Playlist App',
+      selectedArtists: [], 
+      searchedArtists: [],
+      topTracksIds: [],
+      displayPlaylist: 'none',
+      random: 0
+    }
+  }
+
+  componentDidMount() {
+      this.getUser();
+  }
+
+  sliceUserId = (url) => {
```

Comment:
Can we put this method right beneath the getUser method since thats the one that uses it. It keeps our code a bit more sequential.